src/shared/components/DismissibleMessage/DismissibleMessage.style.ts

```diff
@@ -30,9 +31,33 @@ export const MessageTitle = styled(Text)`
   word-break: break-word;
 `
 
+export const MessageButtonsContainer = styled.div`
+  display: flex;
+  margin-left: auto;
+`
+
+export const MessageActionButton = styled(Button)`
+  display: flex;
+  align-items: center;
+  padding: ${sizes(2)};
+  min-width: auto;
+`
+
 export const MessageDescription = styled(Text)`
   margin-top: ${sizes(2)};
   line-height: ${sizes(5)};
   color: ${colors.gray[300]};
   word-break: break-word;
 `
+
+export const MessageWrapper = styled.div<MessageProps>`
+  position: relative;
+  padding: ${sizes(4)};
+  box-shadow: ${({ variant }) => variant === 'primary' && `inset 0px 0px 0px 1px ${colors.gray[700]}`};
```

Comment:
Im getting: ![image](https://user-images.githubusercontent.com/51168865/124463883-caf64400-dd93-11eb-8d3d-8a740c8a38ce.png)

```suggestion
  box-shadow: ${({ variant }) => variant === 'primary' && `inset 0 0 0 1px ${colors.gray[700]}`};
```

If this suggestion introduces any problems you should just add stylelint ignore.